huaweicloud/services/waf/resource_huaweicloud_waf_dedicated_domain.go

```diff
@@ -128,6 +128,7 @@ func ResourceWafDedicatedDomainV1() *schema.Resource {
 			},
 			"tls": {
 				Type:     schema.TypeString,
+				Optional: true,
 				Computed: true,
 			},
```

Comment:
Schemas should be sorted by `Required`, `Optional` and `Computed`, so they should be:

```
	"protect_status": {
		Type:     schema.TypeInt,
		Computed: true,
		Optional: true,
	},
	"tls": {
		Type:     schema.TypeString,
		Optional: true,
		Computed: true,
	},
	"certificate_name": {
		Type:     schema.TypeString,
		Computed: true,
	},
	"access_status": {
		Type:     schema.TypeInt,
		Computed: true,
	},
	"protocol": {
		Type:     schema.TypeString,
		Computed: true,
	},
```

Reply:
done